lib/network/modules/EdgesHandler.js

```diff
@@ -94,7 +94,11 @@ class EdgesHandler {
         }
       },
       selectionWidth: 1.5,
-      selfReferenceSize:20,
+      selfReferenceSize: 20,
+      selfReference: { 
+        size: 20,
+        angle: undefined,
+      },
```

Comment:
Maybe you could set default angle to the top right corner to simplify the logic later on (no need to check if angle is set because it always would).